src/main/java/chess/Piece.java

```diff
@@ -31,11 +32,37 @@ public void setSymbol(final Symbol symbol) {
     public void setStatus(final Status status) {
         this.status = status;
     }
-
-    public void setPosition(final Position position) {
-        this.position = position;
+    public void setPosition(final Position source) {
+        this.position = source;
+    }
+    public void setChessBoard(final ChessBoard chessBoard) {
+        this.chessBoard = chessBoard;
     }
 
+    public boolean move(final Position position, final ChessBoard chess) {
```

Comment:
This method doesn't nees pass a ChessBoard.

Reply:
It is necessary to pass to identify the Chessboard with the actual piece

Reply:
You are right, Change made can you review it? please